### PR TITLE
writing a bubbly file and tutorial for gosec

### PR DESCRIPTION
**Pull request by nate-droid**

### CODE REVIEW

File: docs/docs.go [hunk 1]
```diff
@@ -12,7 +12,335 @@ import (
 	"github.com/swaggo/swag"
 )
 
-var doc = ``
+var doc = `{
```
Comment (jlarfors) [inline docs/docs.go:15]: Assuming this isn't supposed to be here?

**Review by jlarfors — CHANGES_REQUESTED** [review]
Nice work! Only comment about checking in the generated swag docs (clarify if this is supposed to be there)

File: docs/docs.go (hunk 1 above)
Comment (nate-droid) [reply docs/docs.go:15]: Good catch, not sure how that happened. I'll take that out.

Comment (nate-droid) [reply docs/docs.go:15]: I updated the swag docs using `swag init` so should be a bit nicer now.

**Review by benmarsden — APPROVED** [review]

**Review by jlarfors — APPROVED** [review]
LGTM

**Pull request merged**